AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -54,10 +54,9 @@ def reduce_compscan_inf(h5,rfi_static_flags=None,chunks=16,return_raw=False,use_
     stdv = {}
     calibrated = False # placeholder for calibration
     h5.select(compscans=compscan_index)
-    h5.select(reset='B') # Resets only pol,corrprods,ants
-    active_ants = list(set(h5.antlist) & set(find_active_ants(h5, 0.85))) # Only those specified AND active during this compscan
+    active_ants = find_active_ants(h5, 0.85)    # Only those specified AND active during this compscan
```

Comment:
Can you (or @adriaanph) verify that `--exclude-ants` still work with this new setup? Ideally we need a dataset where the active antennas also change from one compscan to the next.

Reply:
this doesn't work, because reduce_compscan_inf() is called in a loop, for each compscan (L350). there are at least two solutions:
1) remove "h5.select(reset='B')" from L57 and move "h5.select(compscans="interf...",ants=ant_list)" from L333 to L350, OR
2) propagate 'ant_list' as an argument into reduce_compscan_inf(antlist=ant_list)


FYI i have used dataset with CBID 1705255287 (all antennas except m047 & m050) as an example to empirically confirm this. m014 gets flagged out in CS3 and then remains flagged out permanently instead of being considered again in CS4.

Reply:
i have committed the change corresponding to my no. 1